.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,54 @@
+name: Integration Test
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  integration_test:
+    runs-on: ubuntu-latest
+    env:
+      GO_VERSION: 1.17
+      HARBOR_VERSION: 2.4.1
+      OCI_IMAGE_NAME: nginx
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v2
+
+    - name: Install Golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.GO_VERSION }}
+
+    - name: Build Accel
+      run: |
+        make install-check-tools
+        make
+
+    - name: Install Docker
+      uses: sudo-bot/action-docker-compose@latest
+
```

Comment:
Why we need to install docker here? Ubuntu runner seems to have it already: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#tools

Reply:
Thanks, fixed.